pkgs/development/python-modules/prometheus-pandas/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, ipython
+, matplotlib
+, numpy
+, pandas
+, requests
+}:
+
+buildPythonPackage rec {
+    pname = "prometheus-pandas";
+    version = "0.3.2";
+
+    src = fetchPypi {
+      inherit pname version;
+      hash = "sha256-zysdlYPjvhextAfW4vvGN6mSPVUbWLV+AvQ1hGWCAbw=";
+    };
+
+    propagatedBuildInputs = [
+      ipython
+      matplotlib
+      numpy
+      pandas
+      requests
+    ];
+    # Tests fail because: "NameError: name 'get_ipython' is not defined"
```

Comment:
The upstream seems to have no tests.

Reply:
Oh. Deleted the comment.

Reply:
Let's leave `doCheck = false` and add a brief comment to show that there is no test.

Reply:
Done.